tests/cpu_bugs/xen_pv.pm

```diff
@@ -0,0 +1,394 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: James Wang <jnwang@suse.com>
+# Author: Di Kong <di.kong@suse.com>
+#
+# This testsuite for the mitigations control on XEN PV guest.
+# And we will integrate HVM control flow here soon.
+#
+# We testing each of parameter. To check the influence of each of them.
```

Comment:
```suggestion
# We testing each parameter. To check the influence each of them.
```
my English skills actually pretty bad , but I have a gut feel that this sounds better ...